app/OwnsModels.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+trait OwnsModels
+{
+    /**
+     * Determine if this model owns the given model.
+     *
+     * @param Model $model
+     *
+     * @return bool
+     */
+    public function owns(Model $model)
+    {
+        return $this->id == $model->{$this->getForeignKey()};
```

Comment:
This is a gotcha for new players; the strict comparison fails because the foreign key is a string unless you explicitly cast it to an `int`, which I've done in this instance (but wouldn't work ordinarily)